src/main/java/uk/gov/justice/digital/nomis/jpa/repository/OffenderExternalMovementsRepository.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.nomis.jpa.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.justice.digital.nomis.jpa.entity.OffenderExternalMovement;
+
+import java.util.Optional;
+
+@Repository
+public interface OffenderExternalMovementsRepository extends JpaRepository<OffenderExternalMovement, Long> {
+
+    Optional<OffenderExternalMovement> findByOffenderBookIdAndMovementSeq(Long movementSeq, Long bookingId);
```

Comment:
are those two fields indexed in the table?

Reply:
yes - there is a unique index